web/src/components/SpeakerCard.vue

```diff
@@ -11,17 +11,17 @@
         <v-row
           class="my-1"
         >
-          <span class="speaker-title">{{ speaker.title }}</span>
+          <span class="speaker-title">{{ job_title }}</span>
```

Comment:
Would it be possible to use camelCase, as it's more common in Vue.js than snake_case? 🐍